packages/terra-spacer/tests/nightwatch/spacer-spec.js

```diff
@@ -0,0 +1,263 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+/* eslint-disable no-unused-expressions */
+const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
+
+module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
+  'Displays a default spacer': (browser) => {
+    browser
+      .url(`${browser.launchUrl}/#/tests/spacer-tests/default`)
+      .expect.element('#spacer').to.be.present;
+  },
+
+  'Displays spacer with very very small margin': (browser) => {
+    browser
+      .url(`${browser.launchUrl}/#/tests/spacer-tests/SpacerSmall2`)
+      .expect.element('#spacermarginsmall-2').to.be.present;
+    browser.expect.element('#spacerpaddingsmall-2').to.be.present;
+    browser.expect.element('#spacermarginpaddingsmall-2').to.be.present;
+
+    browser.expect.element('#spacermarginsmall-2').to.have.css('margin-top').which.equals('2.996px');
```

Comment:
Do we want to test the css values of these margins? I know in the past these can evaluate to different values depending on the browser tested in.

@bjankord

Reply:
I'm fine removing these. Once we add screenshot testing to terra-core, we should have tests which capture this.

Reply:
can just check that we have the css class on all of these

Reply:
Yeah that sounds like a good idea @kschuste.